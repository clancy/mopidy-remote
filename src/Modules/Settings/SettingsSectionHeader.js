import React from 'react';
import { View, Text, StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  container: {
    flex: 1,
    padding: 8,
    justifyContent: 'center',
  },
  text: {
    color: 'white',
    fontWeight: 'bold',
    fontSize: 13,
  }
});

const SectionHeader = (props) => (
  <View style={styles.container}>
    <Text style={styles.text}>{props.text}</Text>
  </View>
);

export default SectionHeader;
