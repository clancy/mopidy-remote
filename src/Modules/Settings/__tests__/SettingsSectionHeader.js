import 'react-native';
import React from 'react';
import SettingsSectionHeader from '../SettingsSectionHeader';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
describe('SettingsSectionHeader tests', () => {
  test('renders SettingsSectionHeader correctly', () => {
    const tree = renderer.create(
      <SettingsSectionHeader
        text={"Text"} 
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
