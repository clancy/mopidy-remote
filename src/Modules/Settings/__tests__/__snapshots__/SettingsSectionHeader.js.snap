exports[`SettingsSectionHeader tests renders SettingsSectionHeader correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "justifyContent": "center",
      "padding": 8,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "white",
        "fontSize": 13,
        "fontWeight": "bold",
      }
    }>
    Text
  </Text>
</View>
`;
